features/karaf/openhab-core/src/main/feature/feature.xml

```diff
@@ -438,6 +450,7 @@
 		<feature>openhab-core-automation-module-script</feature>
 		<feature>openhab-core-automation-module-script-rulesupport</feature>
 		<feature>openhab-core-automation-module-media</feature>
+		<feature>openhab-core-config-discovery-sddp</feature>
```

Comment:
I am pretty sure you can remove this. You just need to include the `openhab-core-config-discovery-sddp` feature in every binding that uses sddp discovery (as you now have in the `openhab-core-config-discovery-addon-sddp` feature).

Reply:
Makes sense.